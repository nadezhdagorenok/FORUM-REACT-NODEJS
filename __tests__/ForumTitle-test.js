import React from 'react';
import renderer from 'react-test-renderer';

import ForumTitle from '../components/ForumTitle';

    test('render ForumTitle', () => {
    const component = renderer.create(
        <ForumTitle title="Flower's Topics!"/>
    );
    let wrap = component.toJSON();
    expect(wrap).toMatchSnapshot();
});
