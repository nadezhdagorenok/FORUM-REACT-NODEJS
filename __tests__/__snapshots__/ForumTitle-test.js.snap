// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ForumTitle 1`] = `
<div
  className="ForumTitle"
>
  <h1>
    Flower's Topics!
  </h1>
</div>
`;
